images/Dockerfile

```diff
@@ -1,3 +1,10 @@
+FROM golang:1.14.2-stretch AS bootstrap-bin
```

Comment:
Is there a reason to use this specific version? Wouldn't `golang:1.14` be enough here?

Reply:
just to make sure that its a debian distro. `golang:1.14` as for today provides that but there is no guarantee for that